cpp/arcticdb/storage/azure/azure_client_wrapper.hpp

```diff
@@ -18,6 +18,28 @@ namespace arcticdb::storage::azure {
 
 static const size_t BATCH_SUBREQUEST_LIMIT = 256; //https://github.com/Azure/azure-sdk-for-python/blob/767facc39f2487504bcde4e627db16a79f96b297/sdk/storage/azure-storage-blob/azure/storage/blob/_container_client.py#L1608
 
+// some common error codes as per https://learn.microsoft.com/en-us/rest/api/storageservices/blob-service-error-codes
+enum AzureErrorCode {
```

Comment:
prefer enum class

Reply:
Fixed in [4ea3442](https://github.com/man-group/ArcticDB/pull/1344/commits/4ea3442882bfdc4001ff117531804a8ba28a8d08)